src/views/Correction.vue

```diff
@@ -350,20 +350,26 @@ export default {
     // if required data isn't set, go back to dashboard
     if (!this.entityIncNo || isNaN(this.correctedFilingId)) {
       this.$router.push({ name: Routes.DASHBOARD })
-    } else {
-      this.dataLoaded = false
-      this.loadingMessage = `Preparing Your Correction`
-      // first fetch original filing, then fetch draft (which may overwrite some properties)
-      await this.fetchOrigFiling()
-      if (this.filingId > 0) {
-        await this.fetchDraftFiling()
-      }
-      this.dataLoaded = true
     }
   },
 
   /** Called when component is mounted. */
-  mounted (): void {
+  async mounted (): Promise<void> {
```

Comment:
I split up created/mounted here same as everywhere else.

The reason is that accessing any children by `$ref` (which is done in the other filings) doesn't work in "created" because the children are not created yet. But it works fine in "mounted".